package.json

```diff
@@ -17,7 +17,7 @@
     "@hapi/lab": "22.x.x"
   },
   "scripts": {
-    "test": "lab -t 97 -a @hapi/code -L",
+    "test": "lab -t 100 -a @hapi/code -L",
```

Comment:
Any reason to bump this to 100 now instead of 99 and bumping it again when we reach 100%?

Reply:
I figured it would be nice to know we can merge #7 when it turns green.